i18n/de.toml

```diff
@@ -29,12 +29,12 @@ greenShareSub1 = "der Leistung kommt"
 greenShareSub2 = "von PV und Speicher"
 power = "Ladeleistung"
 powerSub1 = "{activeClients} von {totalClients} Nutzern"
-powerSub2 = "laden..."
+powerSub2 = "laden …"
```

Comment:
Die kombinierten Punkte bevorzugen wir, das Leerzeichen bewirkt schlimmstenfalls allerdings einen Umbruch. Warum ist das notwendig?

Reply:
@andig Danke für Deine Reaktion.

- Hier ist https://www.duden.de/sprachwissen/rechtschreibregeln/auslassungspunkte anzuwenden.
- Wenn der Zeilenumbruch verhindert werden soll, dann könnte anstelle des Leerzeichens vor den Auslassungspunkten ein geschütztes Leerzeichen (https://de.wikipedia.org/wiki/Gesch%C3%BCtztes_Leerzeichen) gesetzt werden: HTML: `&nbsp;`, Unicode: `U+00A0`

Reply:
Ohne Adblocker leider nicht zu lesen, wenns Konsens gibt (@naltatis) würde ich mich aber anschließen. Dann sollten wir das aber einmalig für alle Sprachen tun.

Reply:
Ich wüsste allerdings nicht, wie wir das non-breaking Space durch Weblate bekommen sollten ohne dass es später wieder rausoptimiert wird?

Reply:
Ich würde spezielle Whitespaces nicht durch Nutzer in Weblate pflegen lassen. Die Fehlerquelle ist dort sehr hoch und in der normalen Ansicht ist das nicht ersichtlich. Hier im konkreten Fall seh ich aber auch kein Problem mit einem normalen Whitespace. Wenns umbricht brichts halt um. Da müssen wir im Layout drauf eingehen, nicht in der Textpflege.

Reply:
Ist für Deutsch nun an allen Stellen in Weblate angepasst.